doc/source/installation.rst

```diff
@@ -65,6 +65,13 @@ available on your controller or host machines.
   highly recommended to upgrade user setuptools even when using a proper
   virtualenv as shown above.
 
+Keep in mind that on selinux supporting systems, if you install into a virtual
+environment, you may face https://github.com/ansible/ansible/issues/34340 even
+if selinux is not enabled or is configured to be permissive.
+
+It is your reponsability to assure that soft dependencies of Ansible are
```

Comment:
```suggestion
It is your responsibility to assure that soft dependencies of Ansible are
```

Reply:
// side note: integrate a spell checker.
The one we use @ aiohttp is based on libenchant: https://github.com/aio-libs/aiohttp/blob/9504fe2/.travis.yml#L54